Dockerfile.stable

```diff
@@ -34,12 +37,25 @@ ADD https://download-installer.cdn.mozilla.net/pub/firefox/releases/$FIREFOX_VER
 ADD https://github.com/mozilla/geckodriver/releases/download/$GECKODRIVER_VERSION/geckodriver-$GECKODRIVER_VERSION-linux64.tar.gz \
     /root/gecko.tar.gz
 
+#copying nginx config for selenium proxy
+ADD ./nginx-selenium.conf /etc/nginx/conf.d
+
+RUN  sed -i -- 's/worker_processes.*;/worker_processes 1;/' /etc/nginx/nginx.conf && \
```

Comment:
Please include in-line comment about the purpose of these lines.

Reply:
Of course. those are just optimization values.  nginx starts 5 workers by default. so, I'm replacing it to 1 because 1 worker is more than enough for proxying requests to one selenium server instance.

Despite that, nginx throws a warning about not enough resources by default. It can be ignored but I decided to get rid of it.

Reply:
to ease archeology, lets move that inline with the code